R/xcmsSet.R

```diff
@@ -744,7 +744,7 @@ setMethod("group.nearest", "xcmsSet", function(object, mzVsRTbalance=10,
         mplenv$peakIdxList <- data.frame(peakidx=which(mplenv$peakmat[,"sample"]==sample),
                                          isJoinedPeak=FALSE)
         if(length(mplenv$peakIdxList$peakidx)==0){
-            cat("Warning: No peaks in sample",s,"\n")
```

Comment:
yes, variable `s` is not defined in the function. This would cause an error if there was no peak in the sample. Alternatively, replace `s` by `sample`, which is the sample index.